src/Components/Alert/Components/Filters/WaysToBuy.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Checkbox, Flex, Join, Spacer, Text } from "@artsy/palette"
+import { entries } from "lodash"
+import { FC } from "react"
+import { WAYS_TO_BUY_OPTIONS } from "Components/ArtworkFilter/ArtworkFilters/WaysToBuyFilter"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { SearchCriteriaAttributeKeys } from "Components/SavedSearchAlert/types"
+
+interface WayToBuy {
+  selected: boolean
+  name: string
+  key: SearchCriteriaAttributeKeys
+}
+
+export const WaysToBuy: FC = () => {
+  const { state, dispatch } = useAlertContext()
+
+  const checkboxes: WayToBuy[] = entries(WAYS_TO_BUY_OPTIONS).map(
+    ([key, value]) => ({
+      key: key as SearchCriteriaAttributeKeys,
+      name: value.name,
+      selected: state.criteria[key],
+    })
+  )
+
+  return (
+    <>
+      <Text variant="sm-display">Ways to buy</Text>
+      <Spacer y={2} />
+      <Box style={{ columns: "2" }}>
+        <Flex flexDirection="column">
```

Comment:
Instead of `<Flex flexDirection="column">` we might want to use Palette's [GridColumns](https://palette.artsy.net/elements/gridcolumns/#gatsby-focus-wrapper) here ([example](https://github.com/artsy/force/blob/6956a455b93aeeedcd7a0d9503ac7f1249d61416/src/Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter.tsx#L53)).


At the moment, both new filters have two columns instead of one on mWeb.
![Screenshot 2023-11-09 at 17 50 00](https://github.com/artsy/force/assets/4691889/8327da7f-9d26-459d-83e3-8fb4b2914c78)

Reply:
Ah, I see now that it was changed, will update it to `GridColumns`  as well 👍